extensions/logs/logs.go

```diff
@@ -54,6 +54,7 @@ type Config struct {
 	WorkerCount          int           `default:"8" desc:"Number of log collector workers" split_words:"true"`
 	MaxKubeConfigs       int           `default:"3" desc:"Number of used kubeconfigs" split_words:"true"`
 	AllowedNamespaces    string        `default:"(ns-.*)|(nsm-system)|(spire)|(observability)" desc:"Regex of allowed namespaces" split_words:"true"`
+	AllowedContainers    []string      `default:"nsc-simple-docker,nse-simple-vl3-docker" desc:"Allowed docker containers" split_words:"true"`
```

Comment:
Could we use here group of regexp patterns? For example, `(nsc-.*)|(nse-.*)|(nsmgr-.*)|(forwarder-.*)`

Reply:
done